frame/lending/src/types.rs

```diff
@@ -26,7 +26,7 @@ pub type MarketId = u32;
 #[repr(transparent)]
 pub struct MarketIndex(
 	#[cfg(test)] // to allow pattern matching in tests outside of this crate
```

Comment:
cfg(test) is not propagated outside of crates btw.

Reply:
oh, fixig. for some reason fmt did not cleaned it.